docs/release-notes/v0.7.3.rst

```diff
@@ -14,6 +14,43 @@ Fixes
   are correctly handled.
   (PRs :pr:`2277`, :pr:`2278`)
 
+  The fixed bug was subtle and only occurred for specific configurations of
+  settings and arguments.
+  To determine if you might have been affected by it see if you used a setup
+  like the following in any part of your code
+
+  .. code:: python
+
+    pyhf.set_backend("jax", pyhf.optimize.minuit_optimizer(strategy=0))
+
+    ...
+
+    fit_result, opt_result = pyhf.infer.mle.fit(
+        data, model, return_result_obj=True, do_grad=False
```

Comment:
Maybe worth explicitly pointing out that `do_grad=False` is another requirement to trigger the bug? I also noticed it doesn't depend on the backend being picked, but of course unlikely that anyone would use `do_grad=False` together with `numpy`.

Reply:
That's a good point. I can also add a comment that it is backend independent.

Reply:
Looks very clear now!